src/ListBox/ListBox.stories.jsx

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/no-multi-comp */
+// import { action } from '@storybook/addon-actions';
```

Comment:
If this is not going to be used, let's just delete it.

Reply:
Oops. I didn't mean to leave that in. Thanks for catching that.